packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -643,8 +649,24 @@ export async function getBookings({
   const bookings = await Promise.all(
     plainBookings.map(async (booking) => {
       // If seats are enabled and the event is not set to show attendees, filter out attendees that are not the current user
-      if (booking.seatsReferences.length && !booking.eventType?.seatsShowAttendees) {
-        booking.attendees = booking.attendees.filter((attendee) => attendee.email === user.email);
+      if (booking.seatsReferences.length) {
+        const attendeeSeat = booking.seatsReferences.find(
+          (seatRef) => seatRef.attendee?.email === user.email
+        );
+
+        if (attendeeSeat && attendeeSeat.attendee) {
+          booking.responses = attendeeSeat?.data?.responses || {};
```

Comment:
In case of seated event the responses property is null, which results in improper booking name when eventName property is present.